parser/sql/dialect/opengauss/src/main/antlr4/imports/opengauss/DDLStatement.g4

```diff
@@ -348,6 +361,7 @@ partitionCmd
 
 alterIndexDefinitionClause
     : renameIndexSpecification | alterIndexDependsOnExtension | alterIndexSetTableSpace | alterTableCmds | indexPartitionCmd
```

Comment:
Can you write the rules one by one? According to the code specification, when there are more than 5 branches, a rule needs to be written one line.